game_eggs/minecraft/java/README.md

```diff
@@ -107,4 +107,8 @@ The official Tekkit Classic modpack. Build factories, automate crafting!
 
 ## VanillaCord
 [VanillaCord](https://github.com/ME1312/VanillaCord)
-VanillaCord adds support for BungeeCord's ip-forwarding option on vanilla Minecraft servers.
\ No newline at end of file
+VanillaCord adds support for BungeeCord's ip-forwarding option on vanilla Minecraft servers.
+
+## Glowstone 
```

Comment:
Please follow the alphabetical order list in the readme

Reply:
@regulad This still needs to get fixed.